nginx/tests/test_unit.py

```diff
@@ -81,11 +82,273 @@ def test_plus_api_v3(check, instance, aggregator):
     total = 0
     for m in aggregator.metric_names:
         total += len(aggregator.metrics(m))
-    assert total == 1189
+    assert total == 1191
 
     aggregator.assert_metrics_using_metadata(get_metadata_metrics(), check_submission_type=True)
     aggregator.assert_metric_has_tag('nginx.stream.zone_sync.zone.records_total', 'zone:zone1', count=1)
     aggregator.assert_metric_has_tag('nginx.stream.zone_sync.zone.records_total', 'zone:zone2', count=1)
+    aggregator.assert_metric_has_tag('nginx.stream.zone_sync.zone.records_total_count', 'zone:zone2', count=1)
```

Comment:
```suggestion
    aggregator.assert_metric_has_tag('nginx.stream.zone_sync.zone.records_total_count', 'zone:zone2', count=1)
    
    aggregator.assert_all_metrics_covered()
    aggregator.assert_metrics_using_metadata(get_metadata_metrics(), check_submission_type=True)
```
Let's add this to all the tests

Reply:
I added `aggregator.assert_metrics_using_metadata(get_metadata_metrics(), check_submission_type=True)`, and I will work on asserting all for `aggregator.assert_all_metrics_covered()`

Reply:
I updated tests to assert all metrics covered. By doing this I was also able to find a few more metrics that should have count versions  😄